.github/workflows/nuget-audit.yml

```diff
@@ -0,0 +1,24 @@
+name: NuGet Audit
+on:
+  workflow_dispatch:
+env:
+  DOTNET_NOLOGO: true
+defaults:
+  run:
+    shell: bash
+jobs:
+  build:
+    name: Linux
+    runs-on: ubuntu-22.04    
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4.2.2
+        with:
+          fetch-depth: 0
```

Comment:
We only need to set this so that MinVer has the entire repo history for versioning calculations. Should be fine to remove this here.